src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -182,6 +183,14 @@ public Snapshot retrieveAvailable(UUID id) {
         return snapshotDao.retrieveAvailableSnapshot(id);
     }
 
+    /** Fetch existing Snapshot object that is NOT exclusively locked.
```

Comment:
Our comment formats in this class appear to be weird...shouldn't this look like:
```
/**
 * Fetch existing Snapshot object that is NOT exclusively locked.
 * @param id in UUID format
 * @return a Snapshot object
 */
```
?